tests/test-data/gen-test-signatures.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+set -ex
+
+test_dir=`dirname $0`
+mkdir -p $test_dir
+cd $test_dir
+
+messages=( msg1 msg2 )
+rsa_key_lengths=( 2048 4096 8192 )
+rsa_padding_algs=( pkcs )
```

Comment:
openssl's CLI doesn't support PSS signatures AFAICT but there's some C in http://openssl.6102.n7.nabble.com/RSASSA-PSS-Signature-Question-td22572.html that should be a good start to making a little single-purpose tool we can use for generating signatures.